### ISSUE TITLE
Announce Celo Integration - Tweet - April 19

### ISSUE THREAD

**Issue by c0ric0ri** [Giveth/giveth-planning#1337]
Celo is ready to go on the dApp, 4 tokens are up now.

More tokens are being added in the next weeks.

We should prepare an announcement tweet to let our community know.

**Comment by c0ric0ri** [comment]
@laurenluz we talked about this in the comm's call yesterday and just wanted to get your input on timing to put out this announcement. What would be the best day in your opinion to have this tweet prepared for?

**Comment by c0ric0ri** [comment]
Ok per @GriffGreen's update speaking with Celo, they would like to announce it on Monday for Earth Day. So let's have this reviewed by Friday.

@laurenluz do you have capacity this week to take this one?

**Comment by c0ric0ri** [comment]
I'll take this to prep but per the TG group I see that we're still in discussion with fundraising, so wouldn't plan on this going out this week @laurenluz otherwise please let me know if something has changed

**Comment by c0ric0ri** [comment]
And to clarify above comments, it is still just 4 tokens: CELO, cUSD, cEUR, cREAL

**Comment by laurenluz** [comment]
Yes just those tokens, and we are waiting on project incentives but the work is done and we have the grant money for the rest.

We should prep this

**Comment by laurenluz** [comment]
I would put it out early next week

**Comment by laurenluz** [comment]
oops need comment access @c0ric0ri

**Comment by c0ric0ri** [comment]
We have some cover images here. I personally like the first one better to use for the first tweet:

![image](https://github.com/Giveth/giveth-planning/assets/106524383/aa7d42d9-fc76-4eb4-a241-9ce8915168a6)

![image](https://github.com/Giveth/giveth-planning/assets/106524383/c32591cb-cb1a-4606-bfbb-74161461e2fa)
